sources/PreferenceInfo.h

```diff
@@ -12,8 +12,10 @@ typedef NS_ENUM(NSInteger, PreferenceInfoType) {
     kPreferenceInfoTypeCheckbox,
     kPreferenceInfoTypeInvertedCheckbox,  // true=checked, false=unchecked. Handy when inverting a checkbox's text, but the user defaults key can't be changed.
     kPreferenceInfoTypeIntegerTextField,
+    kPreferenceInfoTypeUnsignedIntegerTextField,
     kPreferenceInfoTypeStringTextField,
     kPreferenceInfoTypePopup,
```

Comment:
I don't know what you mean exactly here. What about the new value type requires a special warning exactly? Or do you mean that the old one needs the warning? Sorry.